string filename
int32 imageTime
---
bool accepted